packages/psss/src/containers/Panels/WeeklyReportsPanel.js

```diff
@@ -124,38 +130,54 @@ const toCommaList = values =>
     .toUpperCase()
     .replace(/,(?!.*,)/gim, ' and');
 
+const getSyndromeAlerts = (countryData, weekNumber) => {
+  if (countryData.length === 0 || !weekNumber) {
+    return [];
+  }
+
+  return countryData[weekNumber].syndromes.reduce(
+    (statuses, syndrome) => (syndrome.isAlert ? [...statuses, syndrome] : statuses),
+    [],
+  );
+};
+
 export const WeeklyReportsPanelComponent = React.memo(
-  ({ countryData, sitesData, isOpen, handleClose, unVerifiedSyndromes, alerts, handleConfirm }) => {
+  ({ countryData, isOpen, handleClose, weekNumber, unVerifiedSyndromes }) => {
```

Comment:
What do you think about sending through just the country data for the week in question? I realise we're also passing in the weekNumber, so it's not totally necessary, but it's also more info than we need within this component, and means having to pass in an extra key every time we want to access the data within the component.

Reply:
(this would be done in `mapStateToProps`)